docs/board.md

```diff
@@ -0,0 +1,142 @@
+
+### Action items / for discussion (Section to be removed before merging.)
+
+#### To remove:
+
+- `needs-info` (`new` state)
+- `needs-design` (not used)
+- `pinned`
+- `triaged`
+- `invalid`
+- `RHOSC`, `GSOC`
+- Is `has-release-notes` still relevant?
+- Do `wontfix` and `invalid` labels provide any value when the issue is closed and marked as not planned with a comment?
+
+#### Edits
+
+- Rename `area/refactor` to `area/technical-dept`.
+- Rename `testing` to `area/testing`
+
+#### Questions
+
+- Do we need complexity?
+- Do we need `kind/documentation`? (We have a separate project for doc-only cards and don't mention this explicitly on other cards.)
+
+## Grooming process cheat sheet:
+
+## Triage
+
+(=process new cards and make basic categorization)
+
+1. Triaging
+   1. **_Is the card not valid or out of our scope?_** => Politely provide reasoning and close the issue as not planned.
+   2. **_Do we have a related card for this?_** => Link the relevant cards, and add to an epic if applicable. Link and close in favour of a duplicate issue if applicable.
+   3. **_Do we need to get more information from the requester? Is it necessary for a team-member to take a look?_** => Assign a person to continue with the discussion and leave it in the `new` column.
+   4. **_Is the task actionable and we are in general sure what the card is about and which way the solution be chosen?_** => Enhance the title and description if needed and move outside of the `new` column.
+   5. **_Does the issue come from an external person and there is a chance of contributing this?_** => Politely ask if the requester won't be able to contribute this with our help.
+2. Labelling
+   1. **_Can we get a new user or allow a new user to start using a feature? Can this determine for the user if Packit will be used in the future?_** => Add `gain/high` label.
+      **_Is there a workaround or the feature is not significant to the user?_** => use `gain/low`.
+   2. **_Is this affecting many users from a role group (e.g. Fedora maintainers, Upstream developers,...)? Can this bring a significant number of new users?_** => Add `impact/high`, add `impact/low` otherwise.
+   3. Add a `demo` label for tasks that are worth presenting to the team or users.
+   4. Add `area/*`, `kind/*` and other suitable labels if needed.
+   5. Add a deadline if applicable.
+3. Prioritising
+   1. **_Is there a strict deadline? Did we break anything crucial? Are we significantly blocking users?_** => [Refine the card](#Refine) right away and move to the `refined` column.
+   2. **_Is there a `gain/high` and `impact/high` label? Do we need/want to finish this within ~3 months? Is this part of our current high-level plans for the quarter?_** => Move to the `priority-backlog` column.
+   3. **_Is there a workaround? Doesn't the task make a user start/stop using Packit? Or, otherwise. _** => move to the `backlog` column.
+
+## Refine
```

Comment:
this could be linked from the `meetings.md` (from the `Refinement` meeting section) to avoid duplication